frame/vesting/src/lib.rs

```diff
@@ -45,10 +45,12 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 mod benchmarking;
+mod migrations;
 #[cfg(test)]
 mod mock;
 #[cfg(test)]
 mod tests;
+pub mod vesting_info;
```

Comment:
```suggestion
mod vesting_info;
```

You already re-export below